benefits/static/css/styles.css

```diff
@@ -566,36 +584,6 @@ footer.global-footer .footer-links a:active {
 }
 
 @media (max-width: 767px) {
```

Comment:
same comment about non-mobile-first media query

Reply:
These ones and lower are because I have not had a chance to work on radio buttons yet. All the media queries below this are just stuff I have not gotten to.